pkg/config/resource.go

```diff
@@ -317,6 +317,12 @@ type UpdateOperationConfig struct {
 	CustomMethodName string `json:"custom_method_name"`
 }
 
+type AdditionalColumnConfig struct {
+	Name string `json:"name"`
+	JSONPath string `json:"json_path"`
+	Type string `json:"type"`
+}
```

Comment:
nit: Please include tiny comments for the struct and its fields. Those will be included in our generated GoDoc documentation

Reply:
Done